src/components/common/Header.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import city_img from '../../assets/images/cityscape.png';
+import '../../styles/Header.less';
+
+const Header = () => {
+  return (
+    <div className="Hero">
+      <p>STORY SQUAD</p>
+      <img src={city_img} alt="MyCity" />
```

Comment:
a different alt tag might be helpful, Since it's for accessibility, it might seem like it's supposed to be a picture of their exact city instead of just a minimalist 2d city? But that's just something to think about for later